infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -879,6 +880,7 @@ private LogEntry getLogEntry(long address, LogData entry) {
                 .setCodecType(entry.getPayloadCodecType().getId())
                 .setData(ByteString.copyFrom(data))
                 .setGlobalAddress(address)
+                .setEpoch(entry.getEpoch())
```

Comment:
I saw getEpoch() is used by StreamingTask::transform. Say there is a layout change and the log entry is written using the previous epoch, will this be misleading to clients that they might think this data is obsolete?